### DISCUSSION TITLE
Help adding UK Asda as a store

### DISCUSSION THREAD

**Discussion by timbollew** [jef/streetmerchant#2013] [Help]
Hi, I'm not very good with website coding so stuck on this. I'm trying to setup a notification for this link (https://www.google.com/amp/s/amp.direct.asda.com/george/toys-character/gaming/gaming-consoles/playstation5-console/050887006,default,pd.html), looking at other listings on the website and trying to pick up the 'Add to Bag' button but the identifier seems to be unique on each listing..I'm sure it's straightforward but I'm out of my depth.

**Comment by lozbinks** [comment]
@timbollew did you ever work this one out?

**Reply by lozbinks** [reply]
it's ok I've done it now.
